qemu_v8.mk

```diff
@@ -356,6 +396,21 @@ optee-os: optee-os-common
 
 optee-os-clean: optee-os-clean-common
 
+################################################################################
+# Hafnium
+################################################################################
+
+HAFNIUM_EXPORTS = PATH=$(HAFNIUM_PATH)/prebuilts/linux-x64/clang/bin:$(HAFNIUM_PATH)/prebuilts/linux-x64/dtc:$(PATH)
+
+.hafnium_checkout:
```

Comment:
`.hafnium_checkout` needs to be added to `.gitignore`; or better create it under `$(OUT_PATH)`?

Reply:
`$(OUT_PATH)` doesn't seem right, I'd rather keep it in the build or `$(ROOT)` directory. I'll add it to `.gitignore`.